File: week-5-assignment/task2.js
const prompt = require("prompt-sync")();

const num = prompt("Enter a number:");

if (num > 0) {
  console.log("The number entered is positive.");
} else if (num < 0) {
  console.log("The number entered is negative.");
} else {
  console.log("The number entered is zero.");
}

File: week-6-assignment/task2.py
while True:
    user_input = input("Enter a number: ")

    # Input validation
    if user_input == "":
        print("Invalid input. Please enter a number.")
        continue

    # Check for positive, negative, or zero.
    user_input = int(user_input)

    if user_input == 0:
        print("Number is zero.")
    elif user_input < 0:
        print("Number is negative.")
    elif user_input > 0:
        print("Number is positive.")
    else:
        print("Invalid input. Please try again!")

    break
